### DISCUSSION TITLE
when i want to start a course it stops and then it says 'should be repaired'

### DISCUSSION THREAD

**Discussion by NewHollandHyper1401** [Courseplay/Courseplay_FS22#2561] [Q&A]
hello i have a problem with courseplay when i want to start a course it stops and then it says 'should be repaired' it is always there no matter what i do threshing cultivators etc hope for help.
![Screenshot (79)](https://github.com/Courseplay/Courseplay_FS22/assets/134838883/4553a224-a0be-41f1-b15e-7b0898c71597)

**Comment by Tensuko** [comment]
Logfile bitte anhängen.

**Reply by NewHollandHyper1401** [reply]
wie meinst du Logfile bitte anhängen?

**Reply by Tensuko** [reply]
https://github.com/Courseplay/Courseplay_FS22/wiki
Ich weis nicht welcher Version du hast und welche Mods noch da reinpfuschen könnten.
Das muss nicht zwangsläufig von CP kommen.

**Reply by NewHollandHyper1401** [reply]
das ist die version in mod hub 7.2.0.5

**Comment by schwiti6190** [comment]
Versuch mal in den Globalen Einstellungen die Einstellungen mit Reparier irgendwas umzustellen, die ziemlich weit unten ist. Evtl. Musst du dafür den Expertenmodus einstellen.

**Reply by Tensuko** [reply]
Wie beretis weiter oben gefragt, logfile?
Den Link zum Wiki was und wie habe ich dir auch gepostet.

**Reply by NewHollandHyper1401** [reply]
ja sry war dumm hoffe das ist die richtige
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/11602971/log.txt)

**Comment by NewHollandHyper1401** [comment]
es geht wieder keine ahnung wie ich das gemacht habe

**Reply by Tensuko** [reply]
Kann AD Setting sein, kann ein Fehler im Fahrzeug sein, es gibt viele Dinge die sowas auslösen können.
